src/store/setting.ts

```diff
@@ -57,6 +57,7 @@ export const settingStoreState: SettingStoreState = {
   },
   confirmedTips: {
     tweakableSliderByScroll: false,
+    noticeAltPortInfo: true,
```

Comment:
「確認されたTips」の中身なので、「エンジンが代替ポートで起動した」とかが良いかもです。
（trueとfalseが反転しているのが混乱しそうだなと思ったので）
こういうのとかどうでしょう。
```suggestion
    engineStartedOnAltPort: false,
```